services/watson-extension/src/watson_extension/clients/insights/image_builder.py

```diff
@@ -0,0 +1,95 @@
+import abc
+from dataclasses import dataclass
+from typing import Optional
+
+import injector
+
+from watson_extension.clients import ContentSourcesURL
+from watson_extension.clients.identity import AbstractUserIdentityProvider
+from watson_extension.clients.platform_request import AbstractPlatformRequest
+
+
+@dataclass
+class EnableCustomRepositoriesResponse:
+    response: Optional[str] = None
+
+
+class ImageBuilderClient(abc.ABC):
+    @abc.abstractmethod
+    async def enable_custom_repositories(
+        self, version: str
+    ) -> EnableCustomRepositoriesResponse: ...
+
+
+class ImageBuilderClientHttp(ImageBuilderClient):
```

Comment:
Why do we have a `ImageBuilder` vs `ContentSources` ?

Reply:
I was following what we had in rasa. The API name is `ContentSources` while the flows belong to `ImageBuilder`. I guess it makes sense to name these `ContentSources`. :thinking: